src/js/lib/pipelines/llmPipeline.ts

```diff
@@ -57,16 +57,7 @@ export class LLMPipeline {
     return result;
   }
 
-  static castOptionsToString(options: Options) {
-    const castedOptions: Options = {};
-
-    for (const key in options)
-      castedOptions[key] = String(options[key]);
-
-    return castedOptions;
-  }
-
-  getAsyncGenerator(prompt: string, generationOptions: Options = {}) {
+  stream(prompt: string, generationOptions: Options = {}) {
```

Comment:
Why is it added? It should have been a class similar to multinomial_causal_lm.py